src/view/element.js

```diff
@@ -151,6 +160,14 @@ export default class Element extends Node {
 		cloned._classes = new Set( this._classes );
 		cloned._styles = new Map( this._styles );
 
+		// Clone custom properties.
+		cloned._customProperties = new Map( this._customProperties );
+
+		// Clone filler offset method if one exists.
+		if ( this.getFillerOffset ) {
```

Comment:
Don't all elements need this method? The check shouldn't be needed if `Element` class defines this abstract method, because the code outside will try to use it anyway.